bundles/org.openhab.binding.lifx/README.md

```diff
@@ -35,12 +35,13 @@ The following table lists the thing types of the supported LIFX devices:
 
 The thing type determines the capability of a device and with that the possible ways of interacting with it. The following matrix lists the capabilities (channels) for each type:
 
-| Thing Type   | On/Off | Brightness | Color | Color Zone | Color Temperature | Color Temperature Zone | Infrared |
-|--------------|:------:|:----------:|:-----:|:----------:|:-----------------:|:----------------------:|:--------:|
-| colorlight   | X      |            | X     |            | X                 |                        |          |
-| colorirlight | X      |            | X     |            | X                 |                        | X        |
-| colormzlight | X      |            | X     | X          | X                 | X                      |          |
-| whitelight   | X      | X          |       |            | X                 |                        |          |
+| Thing Type   | On/Off | Brightness | Color | Color Zone | Color Temperature | Color Temperature Zone | Infrared | Tile Effects |
+|--------------|:------:|:----------:|:-----:|:----------:|:-----------------:|:----------------------:|:--------:|:------------:|
+| colorlight   | X      |            | X     |            | X                 |                        |          |              |
+| colorirlight | X      |            | X     |            | X                 |                        | X        |              |
+| colormzlight | X      |            | X     | X          | X                 | X                      |          |              |
+| whitelight   | X      | X          |       |            | X                 |                        |          |              |
+| tile         | X      | X          | X     |            | X                 |                        |          | X            |
```

Comment:
It would be more consistent with the other thing types to name this a `tilelight`. It will also help to distinguish between LIFX products if they start selling other (non-lighting) products similar to Philips Hue & IKEA Tradfri.

```suggestion
| tilelight    | X      | X          | X     |            | X                 |                        |          | X            |
```